locations/spiders/inpost_gb.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Iterable
+
+from scrapy import Request, Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.categories import apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class InpostGBSpider(Spider):
+    name = "inpost_gb"
+    item_attributes = {"name": "InPost", "brand": "InPost", "brand_wikidata": "Q3182097"}
```

Comment:
TODO: Check the content of  https://github.com/alltheplaces/alltheplaces/issues/5912

Reply:
Okay, non issue - existing is PL only
https://alltheplaces-data.openaddresses.io/map.html?show=https%3A%2F%2Falltheplaces-data.openaddresses.io%2Fruns%2F2024-08-17-13-32-02%2Foutput%2Fpaczkomat_inpost_pl.geojson